configure

```diff
@@ -1752,20 +1752,22 @@ EOF
                 fi
             fi
 
-            if test $buildarmv6 -eq 1; then
-                check_armv6_intrinsics
+            if [ "${ARCH}" != "aarch64" ]; then
+                if test $buildarmv6 -eq 1; then
+                    check_armv6_intrinsics
 
-                if test $HAVE_ARMV6_INTRIN -eq 1 || test $HAVE_ARMV6_INLINE_ASM -eq 1; then
-                    CFLAGS="${CFLAGS} -DARM_SIMD"
-                    SFLAGS="${SFLAGS} -DARM_SIMD"
+                    if test $HAVE_ARMV6_INTRIN -eq 1 || test $HAVE_ARMV6_INLINE_ASM -eq 1; then
+                        CFLAGS="${CFLAGS} -DARM_SIMD"
+                        SFLAGS="${SFLAGS} -DARM_SIMD"
 
-                    if test $HAVE_ARMV6_INTRIN -eq 1; then
-                        CFLAGS="${CFLAGS} -DARM_SIMD_INTRIN"
-                        SFLAGS="${SFLAGS} -DARM_SIMD_INTRIN"
-                    fi
+                        if test $HAVE_ARMV6_INTRIN -eq 1; then
+                            CFLAGS="${CFLAGS} -DARM_SIMD_INTRIN"
+                            SFLAGS="${SFLAGS} -DARM_SIMD_INTRIN"
+                        fi
 
-                    ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} slide_hash_armv6.o"
-                    ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} slide_hash_armv6.lo"
+                        ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} slide_hash_armv6.o"
+                        ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} slide_hash_armv6.lo"
+                    fi
```

Comment:
We can make this patch a lot shorter if we just force `$buildarmv6` as `0` when targeting AArch64.

Reply:
Updated 👍